Formula/p/protoc-gen-js.rb

```diff
@@ -0,0 +1,35 @@
+class ProtocGenJs < Formula
+  desc "Protocol buffers JavaScript generator plugin"
+  homepage "https://github.com/protocolbuffers/protobuf-javascript"
+  url "https://github.com/protocolbuffers/protobuf-javascript/archive/refs/tags/v3.21.2.tar.gz"
+  sha256 "35bca1729532b0a77280bf28ab5937438e3dcccd6b31a282d9ae84c896b6f6e3"
+  license "BSD-3-Clause"
+  head "https://github.com/protocolbuffers/protobuf-javascript.git", branch: "main"
+
+  depends_on "bazelisk" => :build
+  depends_on "protobuf@21"
```

Comment:
Does this not work with latest protobuf?

Reply:
No. [protobuf-javascript](https://github.com/protocolbuffers/protobuf-javascript) currently only works with protobuf 21. protobuf 22 and later requires abseil and there are some compatibility issues to be fixed.

References:

- https://github.com/protocolbuffers/protobuf-javascript/blob/v3.21.2/WORKSPACE#L5-L9
- https://github.com/protocolbuffers/protobuf-javascript/pull/194
- https://github.com/protocolbuffers/protobuf-javascript/pull/196